Cargo.nix

```diff
@@ -9465,7 +9465,7 @@ rec {
       };
       "stackablectl" = rec {
         crateName = "stackablectl";
-        version = "24.3.5";
+        version = "24.3.6";
```

Comment:
Happy to see this file being updated when Cargo.lock changes :D

Reply:
Yupp, that's really neat!